### PR TITLE
Youtube webpage project pull request for code review

### PR DESCRIPTION
**Pull request by javierbotero**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,723 @@
...
+}
+
+header {
+    height: 50px;
+    background-color: white;
+}
+
+#container {
+    width: 1024px;
+    height: 3000px;
```
Comment (pelzolga123) [inline css/styles.css]: If you remove height value, empty space after footer will disappear.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+        <div id="container">
+            <header>
+                <nav id="nav-container">
+                        <div id="logo-container">
+                                <span id="navigation-toggle">
+                                </span>
+                                <a href="https://youtube.com" title="Youtube Home">
+                                    <span title="Youtube Home" id="logo"></span>
+                                    <span id="logo-ornament">NL</span>
+                                </a>
```
Comment (pelzolga123) [inline index.html]: Image should be embedded inside img tag.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires some changes and descriptive Readme file.

**Comment by pelzolga123** [comment]
## Hi Team!  
#### Your project looks good!
* [x] Image should be embedded inside img tag not added from css
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/navbar_logo_requirement.png)
* [x] Project should have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png)
* [x] Project should have like and dislike icons
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_like_requirement.png)
* [x] 'Share' should be a button.
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png)
* [x] Project should have a descriptive README.md
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png)
* Also this part looks not very good, it would look better if you remove empty space after footer:
image

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack. *Happy coding*

**Comment by shshamim63** [comment]
## Status :red_circle:  [!! Required Changes !!] :red_circle: 
___
Hi @javierbotero  and @ivanid22  :wave: ,
Awesome job :clap: . Really a good job as a first project of this HTML & CSS section. Most of the items are rendering at the place it should. Code looks neat and clean. But there are some issue that are actually blocking to declare this project as complete. 
___
##### Please fix the issues below :point_down: to complete this project.
___
### :spades: [Mandatory Changes] :spades: 
___
1. You have to Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.
image

I have done it. Found some errors. Please fix them.
image

2. The readme is not well descriptive. You can make it more descriptive by answering some questions. Like->
..1. What is this project about?
..2. What have you learned?
..3. Is there any online demo.
..4. Who are the contributors?
You can decorate by using this [link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
___
##### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

**Review by nikasulo — APPROVED** [review]

**Comment by nikasulo** [comment]
# :smile: you've done it! @ivanid22 and @javierbotero 
Great work  handling the errors and creating the readme :+1: . Go ahead and mergeeeeeeeee :rocket: .

**Pull request merged**